### PR TITLE
Add default addons.

### PR DESCRIPTION
**Pull request by armandgrillet**
The addons are not given yet as we don't know which ones should be installed by default.

### CODE REVIEW

File: deployments/1.14.x/addons.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: AddonsDeployment
+metadata:
+  name: defaultaddons-1.14.x
+  labels:
+    kubeaddons.mesosphere.io/name: defaultaddons-1.14.x
```
Comment (timaa2k) [inline deployments/1.14.x/addons.yaml]: We will need at least the following here:
```
spec:
  addons:
```

File: deployments/1.14.x/addons.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: AddonsDeployment
+metadata:
+  name: defaultaddons-1.14.x
```
Comment (timaa2k) [inline deployments/1.14.x/addons.yaml]: I would rename all of them to `default-addons-deployment`. They also would be going into the `kubeaddons` namespace.

Comment (timaa2k) [reply deployments/1.14.x/addons.yaml]: The file name should probably match the name :)

Comment (armandgrillet) [reply deployments/1.14.x/addons.yaml]: Would you add the Kubernetes version to the name or would you prefer all the AddonsDeployment to be called `default-addons-deployment`?

Comment (shaneutt) [reply deployments/1.14.x/addons.yaml]: Would it be more appropriate to name this more precisely according to its intended application?

e.g.:

```suggestion
  name: default-ksphere-addons-1.14.x
```
Or something like that? Just wondering what you think, not a blocker.
```

File: deployments/1.14.x/default-ksphere-addons.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: AddonsDeployment
+metadata:
+  name: default-ksphere-addons-1.14.x
+  labels:
+    kubeaddons.mesosphere.io/name: default-ksphere-addons-1.14.x
+spec:
+  addons:
```
Comment (makkes) [inline deployments/1.14.x/default-ksphere-addons.yaml]:
```suggestion
  addons: []
```
This has to be an empty array.

File: deployments/1.15.x/default-ksphere-addons.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: AddonsDeployment
+metadata:
+  name: default-ksphere-addons-1.15.x
+  labels:
+    kubeaddons.mesosphere.io/name: default-ksphere-addons-1.15.x
+spec:
+  addons:
```
Comment (makkes) [inline deployments/1.15.x/default-ksphere-addons.yaml]:
```suggestion
  addons: []
```
This has to be an empty array.

File: deployments/1.16.x/default-ksphere-addons.yaml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: AddonsDeployment
+metadata:
+  name: default-ksphere-addons-1.16.x
+  labels:
+    kubeaddons.mesosphere.io/name: default-ksphere-addons-1.16.x
+spec:
+  addons:
```
Comment (makkes) [inline deployments/1.16.x/default-ksphere-addons.yaml]:
```suggestion
  addons: []
```
This has to be an empty array.

**Pull request merged**